docs/resources/server/advancements.md

```diff
@@ -0,0 +1,173 @@
+Advancements
+============
+
+Advancements are tasks that can be achieved by the player which may advance the progress of the game. Advancements can trigger based on any action the player may be directly involved in.
+
+Data-Driven Advancements
+------------------------
```

Comment:
This section header can be omitted, and the paragraph within can be included in the leading section.